standalone-metastore/metastore-server/src/main/sql/mysql/upgrade.order.mysql

```diff
@@ -19,4 +19,4 @@
 3.2.0-to-4.0.0-alpha-1
 4.0.0-alpha-1-to-4.0.0-alpha-2
 4.0.0-alpha-2-to-4.0.0-beta-1
-4.0.0-beta-1-to-4.0.0-beta-2
+4.0.0-beta-1-to-4.0.0
```

Comment:
1. yes, as someone might have upgraded to alpha/beta release. The current schema evolution strategy is version-centric. We've tried to improve on that, but haven't received any feedback from HMS experts: https://github.com/apache/hive/pull/4060
2. You are right, since there was no 3.2 release we should drop those scripts.